tests/test_samples.py

```diff
@@ -0,0 +1,190 @@
+import json
+import os
+import re
+import sys
+
+from tempfile import NamedTemporaryFile
+from mock import patch
+from tests.test_base import BaseClientTest
+from tests.test_value_constants import *
+from tests.mock_marserver import MockMarServer
+
+if sys.version_info[0] > 2:
+    import builtins  # pylint: disable=import-error, unused-import
+    from urllib.parse import quote_plus  # pylint: disable=no-name-in-module, import-error, unused-import
```

Comment:
I mentioned this also in https://github.com/opendxl/opendxl-tie-client-python/pull/10#discussion_r202222632. I think the `quote_plus` imports could be removed since the tests do not appear to actually use them.

Reply:
Removed